### ISSUE TITLE
Can't debug with VS Code 1.50.1 and NW v0.49.0

### ISSUE THREAD

**Issue by neumartin** [nwjs/nw.js#7598]
Hi!

Im try to debug with this configurations, but none works:

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch NWJS 1",
            "type": "nwjs",
            "request": "launch",
            "nwjsVersion": "any",
            "runtimeExecutable": "/opt/nw/nw",
            "webRoot": "${workspaceRoot}",
        },
        {
            "name": "Launch NWJS 2",
            "type": "nwjs",
            "request": "launch",
            "nwjsVersion": "any",
            "port": 9223,
            "webRoot": "${workspaceRoot}",
            "runtimeExecutable": "/opt/nw/nw",
        },
        {
            "name": "Launch NWJS 3",
            "type": "nwjs",
            "request": "launch",
            "nwjsVersion": "any",
            "runtimeExecutable": "/opt/nw/nw",
            "runtimeArgs": [
                "${workspaceRoot}/build",
                "--remote-debugging-port=9222"
            ],
            "webRoot": "${workspaceRoot}",
            "sourceMaps": true,
            "port": 9222
        }
    ]
}
```
I get this error:

![image](https://user-images.githubusercontent.com/9909839/96400215-89dae700-11a6-11eb-9201-f04b778abc6f.png)

Thanks in advance!

**Comment by rogerwang** [comment]
This looks not an issue in NW.js. Please report it to VS code. Thanks.

**Comment by neumartin** [comment]
@rogerwang thanks for your reply.
It would be very useful if you explain how to debug at least in the most popular IDE's on your official site, since I have not achieved it with VS Code nor IntelliJ (WebStorm).
Many developers would be very grateful.
Thank you!

**Comment by rogerwang** [comment]
From the information provided, it's expecting `main.js`, which is not mandatory in any NW.js app. So it is an issue in your VS code environment.

**Comment by rogerwang** [comment]
btw, IIRC there is some VS code extension that could help debugging NW.js app in vscode.

**Comment by rogerwang** [comment]
https://github.com/karikera/vscode-nwjs-debug

**Comment by neumartin** [comment]
I tryed this extension, but does not work.
Btw, I have the file main.js in my root with this code:

`
nw.Window.open('index.html', {}, function(win) {
    console.log("Launched!");
    win.showDevTools();
});
`
Maybe a bug in the extension or in VS Code.
I tryed with WebStorm and the debugguer seems to work, but do not stop in breakpoint at console.log("Launched!");
I feel like it's hard to start with NW, but from what I could see it looks better than Electron.
I think getting the debugger to work is worth it.
